packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -587,14 +581,6 @@ function gid(id: string | undefined | null): string | null | undefined {
 function fid(url: string | undefined | null): string | null | undefined {
     const vmBeingRendered = getVMBeingRendered()!;
     if (isUndefined(url) || url === '') {
-        if (process.env.NODE_ENV !== 'production') {
-            if (isUndefined(url)) {
-                logError(
-                    `Undefined url value for "href" or "xlink:href" attribute. Expected a non-empty string.`,
-                    vmBeingRendered
-                );
-            }
-        }
```

Comment:
I got rid of these `logError`s because they seemed pretty low-value (only logged in dev mode, should arguably be the job of accessibility tools like aXe, not LWC), and I couldn't think of a good way to maintain the dev-only logging when the optimization is in place.